gimie/sources/gitlab.py

```diff
@@ -1,14 +1,287 @@
+from __future__ import annotations
+from dataclasses import dataclass
+import requests
+import os
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Union
+from urllib.parse import urlparse
+
+from dotenv import load_dotenv
+from calamus import fields
+from calamus.schema import JsonLDSchema
 from rdflib import Graph
+
 from gimie.sources.abstract import Extractor
 
 
+from gimie.models import Organization, OrganizationSchema, Person, PersonSchema
+
+from gimie.graph.namespaces import SDO
+
+GL_API_rest = "https://gitlab.com/api/v4/"
+GL_API = "https://gitlab.com/api"
+load_dotenv()
+
+
+def send_graphql_query(
```

Comment:
Maybe we could take these functions out from `sources.github` and `sources.gitlab` and add a base_url option to avoid duplication. Maybe something like:
```python
from gimie.sources.helpers import send_graphql_query
send_graphql_query(base_url=GL_GRAPHQL, query=my_query, data=my_data)
```

Reply:
I would keep this for the refactor PR